FAQ.md

```diff
@@ -6,27 +6,37 @@ _The first rule of FAQ is don't bikeshed the FAQ, leave that for
 Please contribute to the FAQ! Found an explanation in an issue or pull request helpful?
 Summarize the question and quote the reply, linking back to the original comment.
 
+## Concepts
 * [Does `dep` replace `go get`?](#does-dep-replace-go-get)
 * [Why is it `dep ensure` instead of `dep install`?](#why-is-it-dep-ensure-instead-of-dep-install)
+* [What is a direct or transitive dependency?](#what-is-a-direct-or-transitive-dependency)
+
+## Configuration
 * [What is the difference between Gopkg.toml (the "manifest") and Gopkg.lock (the "lock")?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
```

Comment:
I almost tried that in the last PR I submitted to arrange the questions, but naming and deciding where each question belongs was too much and I decided to skip it. 👅 

I still see this question (and the *current answer* especially) to fit both the terminology and design decisions aspects.

Anyway, I would argue that `Gopkg.lock` is not a configuration file and the user should not update it manually (except in very limited cases).

I'm not objecting to leaving this here, just trying to explore the alternatives. 😁